Script/AtomicEditor/ui/HierarchyFrame.ts

```diff
@@ -236,7 +236,7 @@ class HierarchyFrame extends Atomic.UIWidget {
 
             // create
             if (id == "menu create") {
-
+                if (!ToolCore.getToolSystem().project) return;  
```

Comment:
You should be able to ToolCore.toolSystem here:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/TypeScript/AtomicWork.d.ts#L204

Sorry these are not obvious, they are in the AtomicWork.d.ts, which has some extensions in it that need to be filtered into the other declaration files by JSBind, somehow